cmd/downloadStunnerCtl.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
```

Comment:
Can we use a shorter name like `getstunner.sh` or similar? It's easier to type. Also not sure about the `cmd/` dir, what if we put this into it's own subdirectory like `cmd/getstunner/getstunner.sh` ?

Reply:
Sure thing! Done in 3405d04